src/main/scala/com/raphtory/core/implementations/pojograph/entities/external/PojoExVertex.scala

```diff
@@ -17,6 +17,8 @@ class PojoExVertex(private val v: PojoVertex,
 
   override def ID() = v.vertexId
 
+  def name(nameProperty:String="name"): String = getPropertyOrElse(nameProperty,ID.toString)
```

Comment:
You could have put this implementation directly into the Vertex trait, any reason not to?

Reply:
I agree with @ljeub-pometry here, we can put quite a few of these directly into the vertex trait so they are available for all storage models :) 

Are you ok to make the change @narnolddd? I can then merge it in.

Reply:
Good call, I will move it over to Vertex Trait and let you know when I've pushed it up. Thanks for checking this out! :)

Reply:
pushed revised version now :) thanks @ljeub-pometry @miratepuffin!